src/runtime/plan/plan-consumer.ts

```diff
@@ -46,7 +46,7 @@ export class PlanConsumer {
   registerSuggestionsChangedCallback(callback) { this.suggestionsChangeCallbacks.push(callback); }
   registerVisibleSuggestionsChangedCallback(callback) { this.visibleSuggestionsChangeCallbacks.push(callback); }
 
-  setSuggestFilter(showAll, search) {
+  setSuggestFilter(showAll?: boolean, search?: string) {
```

Comment:
actually, i don't `showAll` is not optional, please, remove the ?

Reply:
done